.github/workflows/deploy-staging.yml

```
@@ -106,14 +72,14 @@ jobs:
         env:
           IMAGE_TAG: ${{ github.event.pull_request.number }}
         run: |
-          # Build the image
+         #  Build the image
```

I don't think it matters but just on the off chance that it does
```suggestion
          #  Build the image
```